src/DotNetCore.CAP.MongoDB/IDataStorage.MongoDB.cs

```diff
@@ -27,8 +31,12 @@ public class MongoDBDataStorage : IDataStorage
         public MongoDBDataStorage(
             IOptions<CapOptions> capOptions,
             IOptions<MongoDBOptions> options,
-            IMongoClient client)
+            IMongoClient client,
+            IServiceProvider serviceProvider)
```

Comment:
Why not just inject the ISerializer here as well?

Reply:
Done